pkg/pull/pull.go

```diff
@@ -292,6 +292,10 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		}
 
 		for _, prevChart := range prevHelmCharts {
+			if prevChart.Spec.Exclude.BoolVal || prevChart.Spec.Exclude.String() == "1" {
```

Comment:
Feels like there should be Truthy/Falsey methods to avoid having to copy this pattern around.

Reply:
yeah this isn't how I should actually implement it, in retrospect

I should just be using the .String method, for one

Reply:
because .String already does the conversion of true/false into "1"/"0"